docs/examples/Sensor_Platform_Simulation.py

```diff
@@ -0,0 +1,380 @@
+# specify. In this example we set the birth rate to be 0.10, i.e. on any given time step there is a 10% chance of a new
+# target being initiated. We have set the death rate to 0.01, i.e. on any given time step there is a 1% chance that a
+# target will be removed from the simulation.
+#
+# The above setup will provide a case which loosely approximates an air surveillance radar at an airport.
+from stonesoup.simulator.simple import MultiTargetGroundTruthSimulator
+
+# Set a constant velocity transition model for the targets
+transition_model = CombinedLinearGaussianTransitionModel(
+    [ConstantVelocity(0.5), ConstantVelocity(0.5), ConstantVelocity(0.1)])
+
+# Define the Gaussian State from which new targets are sampled on initialisation
+initial_target_state = GaussianState(StateVector([[0], [0], [0], [0], [9000], [0]]),
+                                     CovarianceMatrix(np.diag([2000, 50, 2000, 50, 100, 1])))
+
+groundtruth_sim = MultiTargetGroundTruthSimulator(
+    transition_model=transition_model,  # target transition model
+    initial_state=initial_target_state,  # add our initial state for targets
+    timestep=timedelta(seconds=1),  # time between measurements
+    number_steps=120,  # 2 minute
+    birth_rate=0.10,  # 5% chance of a new target being birthed
+    death_probability=0.01  # 1% chance of a target being killed
+)
+
+# %%
+# Now that we have set up our ground truth simulation we need to add our platform into the simulation capability. This
+# is done using the :class:`~.PlatformDetectionSimulator`. This simulator allows a *list* of platforms to be added into
+# the simulation, when the simulation is processed the platforms are able to make detections of both the ground truth
+# targets and other platforms.
+#
+# In this case we have a single platform, therefore the radar sensor on this platform will only be able to make
+# measurements of the ground truth objects generated by the simulator.
+
+# Import the PlatformDetectionSimulator
+from stonesoup.simulator.platform import PlatformDetectionSimulator
+
+sim = PlatformDetectionSimulator(groundtruth=groundtruth_sim, platforms=[platform])
+
+# %%
+# Creating the Tracker Components
+# -------------------------------
+# As stated above the aim of this example is to show how :class:`~.Platform`, :class:`~.Sensor` and
+# :class:`~.Simulator` work within Stone Soup. We will therefore quickly build an Unscented Kalman Filter which
+# initiates measurements using a simple heuristic initiation and deletes any track where no detection is associated for
+# 2 consecutive time steps. There are a number of tutorials for how to build the tracking components provided in the
+# :ref:`auto_tutorials`.
+
+# Create an Unscented Kalman Predictor
+predictor = UnscentedKalmanPredictor(transition_model)
+
+# Create an Unscented Kalman Updater, note our sensor adds a measurement model to detections
+updater = UnscentedKalmanUpdater(measurement_model=None)
```

Comment:
It's probably worth pausing to mention that we can do measurement_model=None because I don't think that will have come up before

Reply:
agreed, will add some text too alongside a dramatic pause